Apps/Src/Task_FaultState.c

```diff
@@ -0,0 +1,119 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+#include "os.h"
+#include "Tasks.h"
+#include "BSP_Contactor.h"
+#include "BSP_Lights.h"
+#include "Voltage.h"
+#include "BSP_Fans.h"
+#include "Temperature.h"
+#include "Amps.h"
+#include "EEPROM.h"
+#include "BSP_WDTimer.h"
+#include "CLI.h"
+#include "CANbus.h"
+#include "BSP_UART.h"
+#include "config.h"
+
+/*
+ * Note: do not call this directly if it can be helped.
+ * Instead, call an RTOS function to unblock the mutex
+ * that the Fault Task is pending on.
+ */
+void EnterFaultState() {
+    // Turn Contactor Off
+    BSP_Contactor_Init();
+    BSP_Contactor_Off();
+    //Set Fans to full speed
+    BSP_Fans_Init();
+    BSP_Fans_Set(1, 8);
+    BSP_Fans_Set(2, 8);
+    BSP_Fans_Set(3, 8);
+    BSP_Fans_Set(4, 8);
```

Comment:
Just a thought: I don't think that we've invested anything into individual control of the fans, so it may be good to have a function to call which sets them all to a single value, instead of having to index individually from this layer.

Reply:
Also, might be good to replace `8` with `FAN_MAX_SPEED` or some more descriptive constant.

Reply:
Agreed

Reply:
Updated in #428